### DISCUSSION TITLE
Pencil support

### DISCUSSION THREAD

**Discussion by chr0max** [vAmigaWeb/vAmigaWeb#203] [Ideas]
First, mucho gracias for this great port. I was really sad to not play some good old point`n`click adventures on my ipad without Jailbreaks. Thank you. 

The touchmouse works good and the bluetooth mouse too. Since I have this luxury-toy "Apple Pencil" and it is really fun to use, I thought about using it for Adventures. Is it possible to use it like a mouse? 

I think like the surface pens on windows. Right-Click is hold a click.

**Comment by mithrendal** [comment]
I will check and get back to you...

**Reply by chr0max** [reply]
Thank you, Sir! 🤓

**Comment by mithrendal** [comment]
Today I tried with an Apple Pencil on an iPadPro 2022 …

My observations so far

`Input mode = mouse`, Amiga mouse pointer moves only when the pencil hovers over the display i.e. when moves in a distance of ca. 1 cm to the display, when it touches the surface nothing moves anymore. Looks really cool and I did not expect this. Did not manage to press any of the Amiga mouse buttons with the pencil.

`Input mode=mouse touch pad (btn|move|btn)` Amiga mouse pointer moves when I draw into onto the move area of the display surface, touch down the pencil into the two button areas left and right of the move area triggers Amiga mouse button presses. Selecting a menu item from an Amiga pull down menu does not work with the pencil alone as this would need a pencil down on the  the right button area and the move area at the same time… maybe with two pencils ? 


> Right-Click is hold a click.

I did not get that …  can you describe a bit more what this means ?

**Comment by beronk** [comment]
Is it possible to use a real mouse on iphone?

**Reply by mithrendal** [reply]
iPad works, never tried iPhone ... hang on I will ask chatgpt ...

chatgpt says yes. ...will try that tomorrow.

**Reply by beronk** [reply]
I did tried didn't work :(

**Reply by mithrendal** [reply]
Just tried it with this tutorial to connect Magic Mouse v1 with iPhone and it worked.

https://youtu.be/g_V0MFa-yIo?si=Zn4m3B1PtDEuoFdA

After I connected it via Assitive Touch -> pointer devices -> devices

I opened vAmigaWeb on my iPhone and choose mouse in port1. Then I could move the Amiga mouse pointer. right click was working but left click did not work… Maybe this is because Magic Mouse only has one button ? In the tutorial the guy said Magic Mouse version2 does not work and that mouse did not show up under devices on my iPhone either. Magic Mouse V1 did show up. Any other BT mouse should do too. You could try the tutorial too with your mouse and see what it does.

**Comment by mithrendal** [comment]
https://github.com/dirkwhoffmann/vAmiga/discussions/970

**Comment by mithrendal** [comment]
mouse emulation with a pencil is now online
